src/tbb/arena.h

```diff
@@ -397,158 +372,58 @@ class arena: public padded<arena_base>
 
     std::uintptr_t calculate_stealing_threshold();
 
+    unsigned priority_level() { return my_priority_level; }
+
+    int has_request() { return my_total_num_workers_requested; }
```

Comment:
Isn't the method should return `bool`? Otherwise, consider renaming to explicitly return the number, e.g.:
```suggestion
    int num_requested_workers() const { return my_total_num_workers_requested; }
```

Reply:
Should be `bool` because `thread_dispatcher_client::has_request` returns `bool`

Reply:
Done